test/e2e/storage/persistent_volumes-local.go

```diff
@@ -201,12 +201,20 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				}
 				setupStorageClass(ctx, config, &testMode)
 				testVols := setupLocalVolumesPVCsPVs(ctx, config, testVolType, config.randomNode, 1, testMode)
-				testVol = testVols[0]
+				if len(testVols) > 0 {
+					testVol = testVols[0]
+				} else {
+					framework.Failf("Error LocalVolumesPVCsPVs setup")
```

Comment:
```suggestion
					framework.Failf("Failed to get a test volume")
```
IMO： The error message should describe the specific cause of the failure.

Reply:
yes, correct.